ui/app/adapters/mfa-login-enforcement.js

```diff
@@ -0,0 +1,14 @@
+import ApplicationAdapter from './application';
+
+export default class KeymgmtKeyAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  pathForType() {
```

Comment:
From what I understand of pathForType it's when we want to interrupt the conventional URL. Example in the docs is for underscore situations. Why are we using it here? https://guides.emberjs.com/release/models/customizing-adapters/

Reply:
Ember data will attempt to build the url automatically based on the model name which in this case would result in `/v1/mfa-login-enforcements` (pluralizes and dasherizes the name by default) https://guides.emberjs.com/release/models/customizing-adapters/#toc_path-customization.

Reply:
got it, thank you!